main.go

```diff
@@ -28,17 +28,22 @@ type checker interface {
 var (
 	ignoreCGO     bool
 	ignoreEditors bool
+	showHelpText  bool
 )
 
 func main() {
 	flag.BoolVar(&ignoreCGO, "ignore-cgo", false, "")
 	flag.BoolVar(&ignoreEditors, "ignore-editors", false, "")
+	flag.BoolVar(&showHelpText, "help", false, "")
 	flag.Parse()
 
 	var exit int
 	// TODO(jbd): Check operating system requirements.
 	// See https://github.com/golang/go/wiki/MinimumRequirements for
 	// a more comprehensive list.
+	if showHelpText {
```

Comment:
Just override flag.Usage instead of introducing this flag.

flag.Usage = runHelpText